examples/fabric/v4/cloudRouterConnectivity/cloudRouter2wan/terraform.tfvar.example

```diff
@@ -0,0 +1,12 @@
+equinix_client_id      = “”
+equinix_client_secret  = “”
+notifications_type = “ALL”
+notifications_emails = [“example@equinix.com”,“test1@equinix.com”]
+purchase_order_number = “1-323292"
+fcr_uuid = “<fcr-uuid>”
```

Comment:
Awesome standard for example file! Keeping any Equinix details out with a good input placeholder. Good job :)